app/src/common/utils/attributeUtils.js

```diff
@@ -16,7 +16,7 @@
 
 export const getAttributeValue = ({ key, value } = {}) => value || key;
 
-export const formatAttribute = ({ key, value } = {}, divider = ':') => {
+export const attributeFormatter = ({ key, value } = {}, divider = ':') => {
```

Comment:
As for me default value for the divider looks strange here.
Do we really need it?

Reply:
Technically this function can be called without `divider`.
But I suppose we should not use it as is, only through `formatAttribute...` functions.
I'll remove it.